docs/06_tutorial3.md

```diff
@@ -283,12 +290,14 @@ If you notice a discrepancy where the points of your segment are no longer align
 You created your first segment! Be sure to save it using *“File > Save volpkg”*. The point cloud for this segment is stored in `campfire.volpkg/paths/<your-segment-id>/pointset.vcps`. If you want to view the point cloud in MeshLab, you can convert it to an OBJ file using `vc_convert_pointset`:
 
 ```bash
-vc_convert_pointset -i campfire.volpkg/paths/<your-segment-id>/pointset.vcps -o points.obj
+docker run -v c:\campfire:/campfire ghcr.io/educelab/volume-cartographer:edge vc_convert_pointset -i campfire/campfire.volpkg/paths/<your-segment-id>/pointset.vcps -o points.obj
 ```
 
 ### Rendering the segment
 
-In order to see the content on the surface of our segment, we need to flatten and texture the segment using the `vc_render` tool. While the output from `vc_render` is considered a "final result" that can be placed anywhere, our convention is to place all results in a `working` directory within the `.volpkg` so that all data is kept together:
+In order to see the content on the surface of our segment, we need to flatten and texture the segment using the `vc_render` tool. While the output from `vc_render` is considered a "final result" that can be placed anywhere, our convention is to place all results in a `working` directory within the `.volpkg` so that all data is kept together. You can also create this folder in your normal file explorer window
```

Comment:
```suggestion
In order to see the content on the surface of our segment, we need to flatten and texture the segment using the `vc_render` tool. While the output from `vc_render` is considered a "final result" that can be placed anywhere, our convention is to place all results in a `working` directory within the `.volpkg` so that all data is kept together. You can also create this folder in your normal file explorer window.
```
Adds full stop punctuation mark at end of sentence.